packages/webapp/package.json

```diff
@@ -45,6 +45,7 @@
     "i18next-xhr-backend": "^3.2.2",
     "immer": "^9.0.16",
     "jszip": "^3.10.1",
+    "lodash-es": "^4.17.21",
```

Comment:
I'm using this as fallback because some browsers/the testing environment doesn't have an implementation for the new Object.groupBy functionality. As it starts being more used, we should be able to get rid of this eventually.